shared/css/font.scss

```diff
@@ -6,9 +6,13 @@
 // Try to avoid use of inline styles if you're creating new component).
 // Keeping that in mind -
 
-$metropolis-font: 'Metropolis', sans-serif;
+$metropolis-font: 'Metropolis';
+$noto-sans-fonts: 'Noto Sans', 'Noto Sans Arabic', 'Noto Sans Armenian', 'Noto Sans Bengali', 'Noto Sans SC',
+                  'Noto Sans TC', 'Noto Sans Devanagari', 'Noto Sans Georgian', 'Noto Sans Hebrew', 'Noto Sans JP',
+                  'Noto Sans Kannada', 'Noto Sans Khmer', 'Noto Sans KR', 'Noto Sans Myanmar', 'Noto Sans Sinhala',
+                  'Noto Sans Tamil', 'Noto Sans Telugu', 'Noto Sans Thai', 'Noto Sans Thaana';
 
-$main-font: $metropolis-font;
+$main-font: $metropolis-font, $noto-sans-fonts, sans-serif;
```

Comment:
The script [tools/scripts/convertScssToJs.js](https://github.com/code-dot-org/code-dot-org/pull/59782/files#diff-693ecc71cca8b8c59b3b731cb0a295f1a4bfd8ca95571b9e059812a19deac452) required some adjustments to properly parse newline-split lines and combine multiple CSS variables into one.
## Generated `apps/src/fontConstants.js`

### Before
```js
module.exports = {
  "metropolis-font": "'Metropolis'",
  "main-font": "$metropolis-font, $noto-sans-fonts, sans-serif",
};
```

### After
```js
module.exports = {
  "metropolis-font": "'Metropolis'",
  "noto-sans-fonts": "'Noto Sans', 'Noto Sans Arabic', 'Noto Sans Armenian', 'Noto Sans Bengali', 'Noto Sans SC', 'Noto Sans TC', 'Noto Sans Devanagari', 'Noto Sans Georgian', 'Noto Sans Hebrew', 'Noto Sans JP', 'Noto Sans Kannada', 'Noto Sans Khmer', 'Noto Sans KR', 'Noto Sans Myanmar', 'Noto Sans Sinhala', 'Noto Sans Tamil', 'Noto Sans Telugu', 'Noto Sans Thai', 'Noto Sans Thaana'",
  "main-font": "'Metropolis', 'Noto Sans', 'Noto Sans Arabic', 'Noto Sans Armenian', 'Noto Sans Bengali', 'Noto Sans SC', 'Noto Sans TC', 'Noto Sans Devanagari', 'Noto Sans Georgian', 'Noto Sans Hebrew', 'Noto Sans JP', 'Noto Sans Kannada', 'Noto Sans Khmer', 'Noto Sans KR', 'Noto Sans Myanmar', 'Noto Sans Sinhala', 'Noto Sans Tamil', 'Noto Sans Telugu', 'Noto Sans Thai', 'Noto Sans Thaana', sans-serif",
};
```

Reply:
Oh wow, one more reason to avoid using styles in js 😄